.github/workflows/build.yml

```diff
@@ -14,6 +14,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
I think, we only need it in the publish job, right?

Reply:
I also added it here to avoid confusion when staring at the logs. But I saw that the build job doesn't log the version.